modules/tinymce/src/core/main/ts/text/Whitespace.ts

```diff
@@ -1,8 +1,13 @@
 import { Arr, Strings, Unicode } from '@ephox/katamari';
 
+import { ZWSP } from './Zwsp';
+
 const whiteSpaceRegExp = /^[ \t\r\n]*$/;
 
 const isWhitespaceText = (text: string): boolean => whiteSpaceRegExp.test(text);
+
+const isZwsp = (text: string): boolean => new RegExp(`^[${ZWSP}]*$`).test(text);
```

Comment:
Given we're only checking for a single character, not a range as in the whitespace regex, I think we'd be better off using a for loop over the characters in the string and checking `c === zeroWidth` 🤔 

- [ ] consider switching to a `for` loop

Reply:
done